### PR TITLE
Wrong variable name fixed (maybe copy-paste mistake) + used method instead of direct array access

### PR DESCRIPTION
**Pull request by Quix0r**
Changed:
- used `$this->getRequestValue($request, 'foo', <bar>)` instead of `$request['foo'] ?? <bar>`
- fixed wrong variable naming
- changed double-quotes to single
- see https://github.com/friendica/friendica/issues/11631#issuecomment-1196410497

### CODE REVIEW

**Comment by MrPetovan** [comment]
Unit tests are failing.

**Comment by Quix0r** [comment]
````
 [Symfony\Component\Process\Exception\ProcessTimedOutException]
The process "find . -name \*.php -not -path './vendor/*' -not -path './view/asset/*' -print0 | xargs -0 -n1 php -l" exceeded the timeout of 300 seconds.
````
I don't know how to fix this?

**Comment by MrPetovan** [comment]
This timeout is weird during PHP lint, I restarted the CI suite.

**Review by MrPetovan — APPROVED** [review]

File: src/Model/Contact.php [hunk 1]
```diff
@@ -2969,7 +2969,7 @@ public static function removeFollower(array $contact)
 
 		$cdata = self::getPublicAndUserContactID($contact['id'], $contact['uid']);
 
-		DI::notification()->deleteForUserByVerb($contact['uid'], Activity::FOLLOW, ['actor-id' => $cdata['public']]);
+		DI::notification()->deleteForUserByVerb($contact['uid'], Activity::FOLLOW, ['actor-id' => $contact['public']]);
```
Comment (annando) [inline src/Model/Contact.php:2972]: `$cdata` was correct here.

Comment (Quix0r) [reply src/Model/Contact.php:2972]: Ops, I thought this was the case here. :-/

Comment (Quix0r) [reply src/Model/Contact.php:2972]: Okay, so how can I solve the error? `$cdata` was `bool`.

Comment (annando) [reply src/Model/Contact.php:2972]: According to the code it cannot be `false` but an empty array when no contact had been found.

Comment (Quix0r) [reply src/Model/Contact.php:2972]: My latest PR changes it back but it won't fix the bug here.

**Pull request merged**